src/main/java/com/capitalone/dashboard/collector/DefaultSonar6Client.java

```diff
@@ -250,54 +253,54 @@ protected void updateCodeQualityProjectAnalysis(CodeQuality codeQuality, SonarPr
     }
 
     public List<String> retrieveProfileAndProjectAssociation(String instanceUrl,JSONObject qualityProfile) throws ParseException{
-    	List<String> projects = new ArrayList<>();
-    	String url = instanceUrl + URL_QUALITY_PROFILE_PROJECT_DETAILS + qualityProfile.get("key");
-    	try {
-    		JSONArray associatedProjects = this.parseAsArray(url, "results");
-    		if (!CollectionUtils.isEmpty(associatedProjects)) {
-    			for (Object project : associatedProjects) {
-    				JSONObject projectJson = (JSONObject) project;
-    				String projectName = (String) projectJson.get("name");
-    				projects.add(projectName);
-    			}
-    			return projects;
-    		}
-    		return null;
-    	} catch (ParseException e) {
-    		LOG.error("Could not parse response from: " + url, e);
-    		throw e;
-    	} catch (RestClientException rce) {
-    		LOG.error(rce);
-    		throw rce;
-    	}
-    }
-    
```

Comment:
The indentation change makes it really hard to read what has been changed, so unless it is really difficult to read, refrain from making such changes. Especially this method, it seems you have not made any real change? If that is correct, why touch it just for formatting?